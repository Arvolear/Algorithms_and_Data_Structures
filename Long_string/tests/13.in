638874420835877806327917586927
1380024244595283256725892871063