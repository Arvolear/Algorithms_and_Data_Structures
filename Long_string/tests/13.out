2018898665431161063053810457990
741149823759405450397975284136
881662190005281363628112853751524292186781960736490905393401
2